index.html

```diff
@@ -214,8 +214,11 @@ <h3>Concept and Initial Report</h3>
     <select multiple class="city-search"></select>
 
     <div class="no-requirements-toggle-container">
-      <label for="no-requirements-toggle">No parking requirements</label>
-      <input id="no-requirements-toggle" type="checkbox" />
+      <span>No Parking Requirements</span>
+      <label class="switch">
```

Comment:
Is it possible for the text to still be in the label? I think that's important for semantic web iirc.

Reply:
`<label class="switch">` is being used as the oval background of the switch. I switched to a different toggle design that allows the text to be inside the label.